include/z64pause.h

```diff
@@ -20,6 +20,32 @@ typedef enum {
     /* 0x04 */ PAUSE_WORLD_MAP
 } PauseMenuPage;
 
+// The XZ coordinates in which direction each pause page is at
+// e.g. the item page is in the -z direction
+/*
+ *         [item]
+ *   [
+ *   e          x     [
+ *   q       o-->     m
+ *   u       |        a
+ *   i     z v        p
+ *   p                ]
+ *   ]
+ *         [quest]
```

Comment:
im no ascii artist, but this may look better if you spell "equip" and "map" horizontally? at the expense of the whole diagram shifting to the right a bit with the word "equip" pushing it out